org.bonitasoft.engine.exceptions.impl.MyTestException1_0=First Exception
org.bonitasoft.engine.exceptions.impl.MyTestException1_1=My {0} Exception with {1} parameters
org.bonitasoft.engine.exceptions.impl.MyTestException1_2=a {1} lot {3} of {2} {0} {5} {4}
org.bonitasoft.engine.exceptions.impl.MyTestException1_3=Plop plop
org.bonitasoft.engine.exceptions.impl.MyTestException1_4=kikoo
org.bonitasoft.engine.exceptions.impl.MyTestException2_0=To Bonita or not to Bonita, that is the question
org.bonitasoft.engine.exceptions.impl.MyTestException2_1=My 6@|_|54 \\/\\/I7H \\/\\/3ird Chars &й"'(-и_за)=

org.bonitasoft.engine.identity.SUserNotFoundException_0=The user with id={0} and username={1} can''t be found because it does not exists anymore
org.bonitasoft.engine.bpm.services.exceptions.IllegalTaskStateChangeException_0=You can't change the state of task {1} with id {0} and state {2} to the state {3} because it's not supported